mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -452,8 +458,109 @@ properties:
                 - pub_sub
                 - publish_findings_to_cloud_data_catalog
                 - publish_summary_to_cscc
+                - job_notification_emails
+                - deidentify
               allow_empty_object: true
               send_empty_value: true
               properties: []
               description: |
                 Publish findings of a DlpJob to Data Catalog.
+            - !ruby/object:Api::Type::NestedObject
+              name: 'jobNotificationEmails'
+              exactly_one_of:
+                - save_findings
+                - pub_sub
+                - publish_findings_to_cloud_data_catalog
+                - publish_summary_to_cscc
+                - job_notification_emails
+                - deidentify
+              allow_empty_object: true
+              send_empty_value: true
+              properties: []
+              description: |
+                Sends an email when the job completes. The email goes to IAM project owners and technical Essential Contacts.
+            - !ruby/object:Api::Type::NestedObject
+              name: 'deidentify'  
+              exactly_one_of:
+                - save_findings
+                - pub_sub
+                - publish_findings_to_cloud_data_catalog
+                - publish_summary_to_cscc
+                - job_notification_emails
+                - deidentify
+              description: |
+                Create a de-identified copy of the requested table or files.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'cloudStorageOutput'
+                  required: true
+                  description: |
+                    User settable Cloud Storage bucket and folders to store de-identified files.
+                    
+                    This field must be set for cloud storage deidentification.
+                    
+                    The output Cloud Storage bucket must be different from the input bucket.
+                    
+                    De-identified files will overwrite files in the output path.
+                    
+                    Form of: gs://bucket/folder/ or gs://bucket
+                - !ruby/object:Api::Type::Array
+                  name: 'fileTypesToTransform'
+                  description: |
+                    List of user-specified file type groups to transform. If specified, only the files with these filetypes will be transformed.
+
+                    If empty, all supported files will be transformed. Supported types may be automatically added over time. 
+                    
+                    If a file type is set in this field that isn't supported by the Deidentify action then the job will fail and will not be successfully created/started.
+                  item_type: !ruby/object:Api::Type::Enum
+                    name: 'undefined'
```

Comment:
The  name should be fileType?

Reply:
You can click the link in Diff report to check the generated resource in the provider. 

https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-7687-old..auto-pr-7687

`file_type` is not there.